src/app/(nextjs_migration)/components/client/FalseDoorTest.tsx

```diff
@@ -0,0 +1,65 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+"use client";
+
+import Image from "next/image";
+import ShieldIcon from "./assets/shield-icon.svg";
+import styles from "./FalseDoorTest.module.scss";
+import { CloseBtn } from "../../../components/server/Icons";
+import { useEffect, useState } from "react";
+import { useCookies } from "react-cookie";
+import { useL10n } from "../../../hooks/l10n";
+import Link from "next/link";
+
+export const HandleFalseDoorTest = () => {
+  const [cookies, setCookie] = useCookies(["falseDoorDismissed"]);
+  const [shouldShowFalseDoor, setShouldShowFalseDoor] = useState(false);
+
+  const handleDismiss = () => {
+    setCookie("falseDoorDismissed", "true", { path: "/" });
+  };
+
+  useEffect(() => {
+    setShouldShowFalseDoor(!cookies.falseDoorDismissed);
+  }, [cookies.falseDoorDismissed]);
+
+  return (
+    <>{shouldShowFalseDoor && <FalseDoorTest onDismiss={handleDismiss} />}</>
+  );
+};
+
+type FalseDoorTestProps = {
+  onDismiss?: () => void;
+};
+export const FalseDoorTest = (props: FalseDoorTestProps) => {
+  const l10n = useL10n();
+  const waitlistLink =
+    "https://www.mozilla.org/en-US/newsletter/monitor_waitlist";
```

Comment:
Also: We want to sync with marketing to see if they are expecting us to append UTM parameters to the link. We did that for a [previous banner](https://github.com/mozilla/blurts-server/blob/main/src/views/partials/floatingBanner.js#L19) for them to know where traffic is coming from.

In case we only track on our side using GA the request URI should be [part of the event by default](https://support.google.com/analytics/answer/1012243?hl=en) already.

cc @rhelmer